CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [5.16.1](https://github.com/buildkite/elastic-ci-stack-for-aws/tree/5.16.1) (2023-01-20)
```

Comment:
https://github.com/buildkite/elastic-ci-stack-for-aws/tree/5.16.1 is 404, I think it needs a `v` prefix?

Reply:
oh my god i did the same thing again 🤦